password_security/tests/test_res_users.py

```diff
@@ -13,35 +13,35 @@
 @post_install(True)
 class TestResUsers(SavepointCase):
 
-    def setUp(cls):
-        super(TestResUsers, cls).setUp()
-        cls.main_comp = cls.env.ref('base.main_company')
+    def setUp(self):
```

Comment:
I think that the fix is wrong. Instead of changing all `cls` for `self`, you should change `setUp` for `setUpClass`.

This benefits from `SavepointCase` and speeds up tests.

Another option would be to switch for a `TransactionCase` instead.